src/modules/clusters/cluster-list/expansion-failed-tooltip.html

```diff
@@ -0,0 +1,3 @@
+<div>
```

Comment:
I don't favor the idea of creating the one file for one div. I think we can create a directive and pass the value and show it on conditions.

Reply:
@cloudbehlwe are using uib-tooltip-template bootstrap component for it and it accepts html file only. Even if we create a directive for it, we need to have separate html files there too. Can you explain, how directive will help here.

Reply:
Instead of this, `uib-tooltip-html` can be used which don't accept html file input,